graphql/schema/gqlschema.go

```diff
@@ -2316,6 +2330,24 @@ func getFieldsWithoutIDType(schema *ast.Schema, defn *ast.Definition, providesTy
 	return append(fldList, pd)
 }
 
+// This function check if given gql field has multiple language tags
+func isMultiLangField(fld *ast.FieldDefinition, isMutationInput bool) bool {
+	dgDirective := fld.Directives.ForName(dgraphDirective)
+	if dgDirective != nil {
```

Comment:
we should return early here if dgDirective is nil, will reduce indentation below